src/modules/musig/tests_impl.h

```diff
@@ -894,164 +787,158 @@ void sha256_tag_test(void) {
     }
 }
 
-/* Attempts to create a signature for the combined public key using given secret
- * keys and pre_session. */
-void musig_tweak_test_helper(const secp256k1_xonly_pubkey* combined_pubkey, const unsigned char *sk0, const unsigned char *sk1, secp256k1_musig_pre_session *pre_session) {
-    secp256k1_musig_session session[2];
-    secp256k1_musig_session_signer_data signers0[2];
-    secp256k1_musig_session_signer_data signers1[2];
+/* Attempts to create a signature for the aggregate public key using given secret
+ * keys and keyagg_cache. */
+void musig_tweak_test_helper(const secp256k1_xonly_pubkey* agg_pk, const unsigned char *sk0, const unsigned char *sk1, secp256k1_musig_keyagg_cache *keyagg_cache) {
```

Comment:
(Not this PR) This could probably be consolidated with musig_simple_test a lot, right?

Reply:
Perhaps, but when I wrote this code it was really helpful to have a very simple (and not too abstract) test.